filter_plugins/oo_filters.py

```diff
@@ -799,7 +799,8 @@ def oo_image_tag_to_rpm_version(version, include_dash=False):
             Ex. v3.2.0.10 -> -3.2.0.10
         """
         if not isinstance(version, basestring):
-            raise errors.AnsibleFilterError("|failed expects a string or unicode")
+            raise errors.AnsibleFilterError(
+                "| {} failed expects a string or unicode".format(version))
```

Comment:
We *could* always drop python 2.6 support, RHEL 7 and the latest versions of OS X ship with 2.7. I don't see an issue with dropping support for RHEL 6 clients (especially since we only ship a package for RHEL 7 anyway)